service/controller/v3/version_bundle.go

```diff
@@ -12,6 +12,11 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Added support for setting the chartconfig CR status with the Helm release status.",
 				Kind:        versionbundle.KindAdded,
 			},
+			{
+				Component:   "chart-operator",
+				Description: "Added support for providing values from secrets to helm charts.",
+				Kind:        versionbundle.KindAdded,
```

Comment:
Nitpick but I think casing of Helm should match the other changelog entry. From fmt I think its supposed to be lower case. But I don't care as long as it matches ;)